### PR TITLE
Adding Breadcrumbs UI

### LINKED ISSUE #160: Make breadcrumbs clickable
In this case, "Muni", "18-46th Avenue", etc should be clickable and take you to the appropriate view. 

Screenshot 2019-07-10 at 7 11 16 PM

### PR DESCRIPTION
**Pull request by ckrajewski**
## Proposed changes

Taking @youngj changes and adding UI  portion (breadcrumbs)

Fixes #160

## Screenshot

image

## Link to demo, if any

...

### CODE REVIEW

File: frontend/src/screens/RouteScreen.jsx [hunk 1]
```diff
@@ -70,14 +128,13 @@ function RouteScreen(props) {
...
-      <Paper>
-        <Box p={2} className="page-title">
-          {selectedRoute ? ` ${selectedRoute.title}` : null}
-          {direction ? ` > ${direction.title}` : null}
-          &nbsp;
-          {startStopInfo ? `(from ${startStopInfo.title}` : null}
-          {endStopInfo ? ` to ${endStopInfo.title})` : null}
-        </Box>
+      <Paper className={breadCrumbsWrapper}>
+         <Breadcrumbs separator={ <NavigateNextIcon fontSize="medium"  className={breadCrumbStyling}/> }>
```
Comment (JuanCaicedo) [inline frontend/src/screens/RouteScreen.jsx]: I think one of the warnings @exxonvaldez mentioned could be fixed like this 😀
```suggestion
         <Breadcrumbs separator={ <NavigateNextIcon fontSize="default"  className={breadCrumbStyling}/> }>
```

Comment (ckrajewski) [reply frontend/src/screens/RouteScreen.jsx]: 👍 :)

**Comment by ckrajewski** [comment]
@exxonvaldez 
Sorry for not doing the quick fixes earlier.
Looks like those React warnings are gone for now
image

**Comment by hathix** [comment]
I think Chris is out today, so why don't you just merge Terence

**Pull request merged**